sentry/src/main/java/io/sentry/clientreport/ClientReportKey.java

```diff
@@ -0,0 +1,40 @@
+package io.sentry.clientreport;
+
+import java.util.Objects;
+
+final class ClientReportKey {
+  private final String reason;
+  private final String category;
+
+  ClientReportKey(String reason, String category) {
+    this.reason = reason;
+    this.category = category;
+  }
+
+  public String getReason() {
+    return reason;
+  }
+
+  public String getCategory() {
+    return category;
+  }
```

Comment:
Let's add the nullability annotations

Reply:
Already did, seems like my push didn't go through while on the train.